src/invidious/routes/captcha.cr

```diff
@@ -0,0 +1,8 @@
+module Invidious::Routes::Captcha
```

Comment:
could you please put that under the `Login` route, as it is related to Google log-in?